lib/env.js

```diff
@@ -19,4 +24,17 @@ module.exports = function load_environment_variables_from_env_json (filepath) {
     console.log(msg);
     return msg;
   }
+};
+
+function get_dot_env (filepath) {
```

Comment:
do we have a test+method for when the .env file has a `#` (comment) or at least have this feature as an `issue` we can add in the next iteration?